src/services/completions.ts

```diff
@@ -183,6 +185,20 @@ namespace ts.Completions {
         return { isGlobalCompletion: false, isMemberCompletion: false, isNewIdentifierLocation: false, entries };
     }
 
+    function specificKeywordCompletionInfo(keywords: readonly SyntaxKind[]): CompletionInfo {
+        return {
+            isGlobalCompletion: true,

```

Comment:
I tried making this `false` but the snippets for `for` loops and stuff still showed up 🤔

Reply:
Should it be `false` then?

@mjbvz any idea why that might be happening?

Reply:
Set to `false` now despite seeing no difference locally